docs/concepts/copilot/copilot-data-privacy.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Data Copilot data privacy
+id: copilot-data-privacy
+description: The AI assistant data privacy for data Pipelines and models
+sidebar_position: 11
+tags:
+  - concepts
+  - copilot
+  - generativeai
+  - data privacy
+---
+
+The Prophecy team employs top-notch industry practices to safeguard the security of their application and maintain the privacy of customer data. Below are just a few components of our comprehensive security strategy and system structure:
+
+- Prophecy **does not** store or send anything from your data plane to any third-party LLM providers. Instead, Prophecy makes use of rich metadata to construct the knowledge graph. As a result, Data Copilot can interface with LLM providers while maintaining the privacy of the data itself.
+- Prophecy IDE is hosted on secure servers on AWS. All storage systems are encrypted, and all servers are tightly access controlled and audited. Data is encrypted in transit at all times.
+- Alternatively, Prophecy’s IDE can be installed within an Enterprise network as desired.
+- Prophecy’s IDE accesses your environment through a single IP address dedicated to you, allowing you to protect access to your data resources at the network level. The credentials are stored per user, and only a fully authenticated user can access their environment.
+- An annual penetration test is performed to validate Prophecy’s posture and identify vulnerabilities. Our latest penetration test report was issued in November 2022.
```

Comment:
Annual, huh?

Reply:
@alexanderahn I suggest checking with Vishal to see if we can say "Our latest penetration test report was issued January 2024."
![image](https://github.com/SimpleDataLabsInc/prophecy-docs/assets/20563764/36f3b05e-4eb9-4521-89d1-091d24e31cbf)

Reply:
Yeah, I created this task https://app.asana.com/0/1207656490770058/1207682004998468 last week for this. I'm not sure if it's in the right place.

Reply:
Ack.

I've also added a task to update this later if needed.